src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
-                o.MutationLevel == MutationLevel.Advanced), It.IsAny<IEnumerable<LogMessage>>()));
+            inputs.DevModeInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
         [Theory]
-        [InlineData("--dashboard-compare", "--dashboard-version project")]
-        [InlineData("-compare", "-version project")]
-        public void ShouldEnableDiffCompareToDashboardFeatureWhenPassed(params string[] argName)
+        [InlineData("--concurrency")]
+        [InlineData("-c")]
+        public void WithMaxConcurrentTestrunnerArgument_ShouldPassValidatedConcurrentTestrunnersToStryker(string argName)
         {
-            StrykerOptions options = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.3);
-
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => options = c)
-                .Returns(runResults)
-                .Verifiable();
-
-            var target = new StrykerCLI(mock.Object);
-
-            target.Run(argName);
+            target.Run(new string[] { argName, "4" });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            options.CompareToDashboard.ShouldBeTrue();
+            inputs.ConcurrencyInput.SuppliedInput.Value.ShouldBe(4);
         }
 
         [Theory]
-        [InlineData("--dashboard-compare", "--dashboard-version project")]
-        [InlineData("-compare", "-version project")]
-        public void ShouldEnableDiffFeatureWhenDashboardComparePassed(params string[] argNames)
+        [InlineData("--break")]
+        [InlineData("-b")]
+        public void WithCustomThresholdBreakParameter_ShouldPassThresholdBreakToStryker(string argName)
         {
-            StrykerOptions options = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.3);
-
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => options = c)
-                .Returns(runResults)
-                .Verifiable();
-
-            var target = new StrykerCLI(mock.Object);
-
-            target.Run(argNames);
+            target.Run(new string[] { argName, "20" });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            options.DiffEnabled.ShouldBeTrue();
+            inputs.ThresholdBreakInput.SuppliedInput.ShouldBe(20);
         }
 
         [Theory]
-        [InlineData("--dashboard-url https://www.example.com/")]
-        [InlineData("-url https://www.example.com/")]
-        public void ShouldOverwriteDefaultDashboardUrlWhenPassed(string argName)
+        [InlineData("--mutate")]
```

Comment:
Also not descriptive enough imo, I would suggest `--files-to-mutate`.

Reply:
This used to be `--files-to-exclude` but we deprecated that 😅. I think the reason to go for `--mutate` was that we also support linespans like `--mutate "**/Person.cs:40:0:45:0"` and exclude files.  But `--files-to-mutate` also covers this

Reply:
The reason for mutate is also consistency with sister projects.

Reply:
Wow, didn't know you can specify the lines, cool!